tests/appsec/iast/test_iast_sql_injection.py

```diff
@@ -19,7 +19,7 @@ class TestIastSqlInjection:
 
     EXPECTATIONS = {
         "java": {"LOCATION": "com.datadoghq.system_tests.springboot.iast.utils.SqlExamples"},
-        "nodejs": {"LOCATION": "/usr/app/iast.js"},
+        "nodejs": {"LOCATION": "iast.js"},
```

Comment:
will it fail in nightly builds? Nightly builds test the latest snapshot and the latest release.